src/libpriv/rpmostree-core.c

```diff
@@ -1269,6 +1270,36 @@ rpmostree_context_prepare_install (RpmOstreeContext    *self,
       hy_goal_install (goal, pkg);
     }
 
+  { GPtrArray *repos = dnf_context_get_repos (hifctx);
```

Comment:
As far as journal vs stdout...it's a super messy topic right now, and one we need to improve...but I'd like to get more information into the journal today for the client side so we can more easily debug things there.  Treecompose is already full of junk to stdout, so more/less doesn't matter too much I'd say.